source/common/umf_helpers.hpp

```diff
@@ -243,6 +251,21 @@ static inline auto poolMakeUniqueFromOps(umf_memory_pool_ops_t *ops,
       UMF_RESULT_SUCCESS, pool_unique_handle_t(hPool, umfPoolDestroy)};
 }
 
+static inline auto
+poolMakeUniqueFromOps_CudaProvider(umf_memory_pool_ops_t *ops,
```

Comment:
there is nothing CUDA-specific in this function - it can simply be called ‘poolMakeUniqueFromOps’.

Reply:
There is already the `poolMakeUniqueFromOps()` function above that uses a unique custom provider. This one uses always the CUDA provider. Could you propose another name?

Reply:
@ldorau you don;t need this function, just use makeDisjointPool or makeProxyPool from https://github.com/oneapi-src/unified-runtime/blob/main/source/common/ur_pool_manager.hpp#L307

Reply:
@bratpiorka Renamed to `poolMakeUniqueFromOpsProviderHandle()`.
@igchor Both of those functions take a unique pointer of a provider and return a unique pointer of a pool, but I prefer to use a normal handle in both cases.